models/DataObject/ClassDefinition/Helper/OptionsProviderResolver.php

```diff
@@ -32,9 +32,18 @@ class OptionsProviderResolver extends ClassResolver
 
     public static function resolveProvider(?string $providerClass, int $mode): ?object
     {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '11.1',
+            'Implementing %s is deprecated, use %s instead',
+            MultiSelectOptionsProviderInterface::class,
+            SelectOptionsProviderInterface::class,
+        );
```

Comment:
This needs to be `11.2` now, since `11.1` has already been released.

```suggestion
        trigger_deprecation(
            'pimcore/pimcore',
            '11.2',
            'Implementing %s is deprecated, use %s instead',
            MultiSelectOptionsProviderInterface::class,
            SelectOptionsProviderInterface::class,
        );
```